Probesys/bileto
# Imp/readme

## Related issue(s)



https://github.com/Probesys/bileto/issues/238

## Changes



- add more information to the README + a screenshot
- change the description sentence on the about page
- add a link to the website on the about page

## How to test manually



- go to the README → check it looks fine
- go to the about page → check it looks fine

## Checklist
